pkgs/applications/graphics/rnote/default.nix

```diff
@@ -54,6 +57,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     alsa-lib
+    cmake
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.

Reply:
It was when I was testing it. Some compile step moaned about an executable not being found while looking for cmake. Adding it to build inputs fixed it.
I can try and reproduce the problem if you want…

Reply:
Meson prints that CMake is not found but it only looks for it in case it is needed, even thought that is not the case most of the time. If it is actually needed, it should go to `nativeBuildInputs`.

Reply:
Great, thanks for chiming in!

Reply:
It still should go to `nativeBuildInputs`, though.